indexer-app/src/cmr/indexer/data/concepts/variable.clj

```diff
@@ -10,6 +10,27 @@
    [cmr.indexer.data.elasticsearch :as es]
    [cmr.transmit.metadata-db :as mdb]))
 
+(defn- measurement-quantity->elastic-doc
+  "Returns the elastic document for the given measurement quantity"
+  [quantity]
+  (let [value (:Value quantity)]
+    {:quantity value
+     :quantity.lowercase (util/safe-lowercase value)}))
```

Comment:
Is this measurement-quantity or measurement-identifier-quantity?

Reply:
Not following your question. This a field internally to Elasticsearch. The UMM field is called `MeasurementQuantities`.

Reply:
Never mind. Just a little confused about the naming. Looked at the schema. all good now. Thanks.